v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/transforms/AvroDynamicTransform.java

```diff
@@ -65,6 +65,8 @@ public class AvroDynamicTransform
 
   private static final Logger LOG = LoggerFactory.getLogger(AvroDynamicTransform.class);
 
+  private static String kafkaKeyField = "_key";
```

Comment:
make this final?

Reply:
Done, also in AvroTransform and BigQueryAvroUtils